### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by chucklorenz**
expanded to encompass commands of newly released CLI
incorporates feedback from T Greenfield and C Steele
fixes #1134

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,165 +1,261 @@
...
+
+2. To report a brief description of a specific command:  
+`adapt help create`  
+ 
+<div float align=right><a href="#top">Back to Top</a></div>
+
+### adapt create  
+
+##### Model:  
+`adapt create <type> <path> [<branch>]`
```
Comment (tomgreenfield) [inline README.md]: `<type>` and `<path>` are also optional here.

File: README.md [hunk 2]
```diff
@@ -1,165 +1,261 @@
...
-- 0.0.5  - Added adapt.json (dependency list)
-- 0.0.3  - Added uninstall command
-...
+The Adapt community maintains the [Adapt Plugin Browser](https://www.adaptlearning.org/index.php/plugin-browser/) as a convenient registry of components, extensions, themes, and menus. The plug-in system is powered by [Bower](http://bower.io/): http://adapt-bower-repository.herokuapp.com/packages/. To register, a plug-in must be a valid bower package with *bower.json*, and have a unique name that is prefixed with "adapt-".  
+
+See [Developing plug-ins](https://github.com/adaptlearning/adapt_framework/wiki/Developing-plugins) for more information on defining your plug-in's package and on [registering your plug-in](https://github.com/adaptlearning/adapt_framework/wiki/Registering-a-plugin).  
+<div float align=right><a href="#top">Back to Top</a></div>  
+
+----------------------------
+**Version number:**  2.1.2   <a href="https://community.adaptlearning.org/" target="_blank"><img src="https://github.com/adaptlearning/documentation/blob/master/04_wiki_assets/plug-ins/images/adapt-logo-mrgn-lft.jpg" alt="adapt learning logo" align="right"></a>
```
Comment (tomgreenfield) [inline README.md]: Please bump to [latest version](https://github.com/adaptlearning/adapt-cli/blob/master/package.json#L3).

File: README.md [hunk 3]
```diff
@@ -1,165 +1,261 @@
...
-- 0.0.3  - Added uninstall command
-...
+The Adapt community maintains the [Adapt Plugin Browser](https://www.adaptlearning.org/index.php/plugin-browser/) as a convenient registry of components, extensions, themes, and menus. The plug-in system is powered by [Bower](http://bower.io/): http://adapt-bower-repository.herokuapp.com/packages/. To register, a plug-in must be a valid bower package with *bower.json*, and have a unique name that is prefixed with "adapt-".  
+
+See [Developing plug-ins](https://github.com/adaptlearning/adapt_framework/wiki/Developing-plugins) for more information on defining your plug-in's package and on [registering your plug-in](https://github.com/adaptlearning/adapt_framework/wiki/Registering-a-plugin).  
+<div float align=right><a href="#top">Back to Top</a></div>  
+
+----------------------------
+**Version number:**  2.1.2   <a href="https://community.adaptlearning.org/" target="_blank"><img src="https://github.com/adaptlearning/documentation/blob/master/04_wiki_assets/plug-ins/images/adapt-logo-mrgn-lft.jpg" alt="adapt learning logo" align="right"></a>   
+**Framework versions:**  >=1
```
Comment (tomgreenfield) [inline README.md]: Suggest this line is removed.

Comment (chucklorenz) [reply README.md]: @tomgreenfield Which line to remove? This one?: \***Framework versions:\**  >=1

File: README.md (hunk 1 above)
Comment (chucklorenz) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (chucklorenz) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (chucklorenz) [reply README.md]: done

**Comment by chris-steele** [comment]
@chucklorenz thanks very much for doing this

**Comment by chris-steele** [comment]
@tomgreenfield thanks for reviewing

**Pull request merged**